locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -836,6 +910,52 @@ protected JcloudsSshMachineLocation obtainOnce(ConfigBag setup) throws NoMachine
         }
     }
 
+    private void waitForWinRmAvailable(final NodeMetadata node, final LoginCredentials expectedCredentials, ConfigBag setup) {
+        // TODO: Reduce / remove duplication between this and waitForReachable
+        String waitForWinRmAvailable = setup.get(WAIT_FOR_WINRM_AVAILABLE);
```

Comment:
Perhaps worth merging the `WAIT_FOR_WINRM_AVAILABLE` and the `WAIT_FOR_SSHABLE`. They are conceptually the same thing. Would probably have to deprecate the old key, and generalise the name.